scripts/ci/gitlab/check-each-crate.sh

```diff
@@ -37,7 +39,9 @@ fi
 group=1
 for ((i=0; i < crates_total; i += crates_per_group)); do
   if [ $group -eq "$target_group" ]; then
-    for crate in "${workspace_crates[@]:$i:$crates_per_group}"; do
+    crates_in_group=("${workspace_crates[@]:$i:$crates_per_group}")
+    printf "crates in the group: ${crates_in_group[*]}" >/dev/null # >/dev/null due to "set -x"
+    for crate in "${crates_in_group[@]}"; do
```

Comment:
Just as note: You only check the default features. There is still the possibility that just some features will fail.

Reply:
This was already mentioned in the initial pr.

Reply:
There's a separate ticket for the features: https://github.com/paritytech/substrate/issues/12711